SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -949,7 +974,7 @@ void USpatialReceiver::OnComponentUpdate(Worker_ComponentUpdateOp& Op)
 	}
 }
 
-void USpatialReceiver::HandleUnreliableRPC(Worker_ComponentUpdateOp& Op)
+void USpatialReceiver::HandleRPC(Worker_ComponentUpdateOp& Op)
```

Comment:
should be const

Reply:
Changed to const Worker_ComponentUpdateOp&.